gridmenuview/src/main/java/com/bcc/gridmenuview/utils/ShapeElement.java

```diff
@@ -0,0 +1,6 @@
+package com.bcc.gridmenuview.utils;
+
+public enum ShapeElement {
+    SQUARE,
+    CIRCLE
```

Comment:
Swap the order so the default is the circle one. That way, we don't break existing usage.